code/src/components/app-dropdown/AppDropdown.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { screen, fireEvent } from "@testing-library/react";
+import AppDropdown from "@/components/app-dropdown/AppDropdown";
+import { renderWithProviders } from "@/utils/test-utils";
+
+const mockAppDropdownOnSelect = jest.fn();
+
+const options = [
+  { value: "option1", label: "Label 1" },
+  { value: "option2", label: "Label 2" }
+];
+
+describe("AppDropdown block", () => {
+  beforeEach(() => {
+    renderWithProviders(
+      <AppDropdown
+        options={options}
+        onSelect={mockAppDropdownOnSelect}
+        defaultLabel="Select an option"
+        className="test-dropdown"
```

Comment:
Classname should not be required here

Reply:
Why can't you just remove it then? It reduces amount of code

Reply:
> Why can't you just remove it then? It reduces amount of code

This component get `className` as a prop. And I covered it with the tests ?
Why is it a problem and I should to remove it ? Don't see the reason for it

Reply:
Please remove it

Reply:
> Please remove it if it is redundant

I'm not agree with you. I covered all props value with tests as well. Why is it redundant ? Please explain

Reply:
Okay, let's keep it for now. In the future please try to clean up your redundant code after developing to make our code more readable and less polluted